src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -933,63 +961,49 @@ bool FancyZones::OnSnapHotkey(DWORD vkCode) noexcept
 
 void FancyZones::MoveSizeStartInternal(HWND window, HMONITOR monitor, POINT const& ptScreen, require_write_lock writeLock) noexcept
 {
-    // Only enter move/size if the cursor is inside the window rect by a certain padding.
-    // This prevents resize from triggering zones.
-    RECT windowRect{};
-    ::GetWindowRect(window, &windowRect);
-
-    const auto padding_x = 8;
-    const auto padding_y = 6;
-    windowRect.top += padding_y;
-    windowRect.left += padding_x;
-    windowRect.right -= padding_x;
-    windowRect.bottom -= padding_y;
-
-    if (PtInRect(&windowRect, ptScreen) == FALSE)
+    if (!IsCursorTypeIndicatingSizeEvent())

```

Comment:
nit: perhaps reverse the `if` and do an early return here

Reply:
I intentionaly set this logic that way as I think early-return is less readable.
Could you point out why you think early-return is better option?

Reply:
Also for me the early-return is _more_ readable. Also, we are already using early-returns everywhere, even in this function, look at line 971.

Reply:
@PrzemyslawTusinski mainly two reasons:
- Deeply nested code makes it harder to review/see diffs of the code with git clients/on github/having windows with a vertical split in the editor etc.
- Perceived code complexity increase, more eye-tracking of those deeply-nested brace pyramids. Introducing a new scope implies that the `if` condition is immediately important to the scope's contents, even though we usually just check some preconditions/initialize and error-check helper variables/system API calls for the "main logic" of the function. So if we do this mental split of initialization-preparation and domain logic and do early returns, at the time we're studying that main logic and see an `if` expression, we know that it's directly related to the domain logic itself, not just some unrelated `std::optional` extraction. Having a single `if` pyramid could intertwine that stuff. (this is all purely subjective of course 🙂)